MicrosoftTeams.schema.json

```diff
@@ -447,6 +447,20 @@
                 ]
             }
         },
+        "devicePermissions": {
+            "type": "array",
+            "description": "Specifies the devices the app requests from users.",
```

Comment:
Is this description accurate? The wording seems strange: what does it mean to "request a device".
Perhaps "Specifies device capabilities for which the app can request access"?

Reply:
How about "Specifies the devices that the app can request access from the user"?

Reply:
Ran this by, Jon, our writer. Updating string based on his recommendation.